bug_report.php

```diff
@@ -74,12 +75,29 @@
 require_api( 'string_api.php' );
 require_api( 'user_api.php' );
 require_api( 'utility_api.php' );
+require_api( 'api_token_api.php' );
 
-form_security_validate( 'bug_report' );
+# Check if this is a usual submission via UI or scripted autosubmission
```

Comment:
We can use the REST API (see #1037) which I just submitted for review.  My goal is to get this in and add methods gradually.  Your feedback is welcome there.  Using the bug_report.php is not a path I would take.

I would be totally happy to have the new API added by this PR only show up in the REST API.